pytools/dot.py

```diff
@@ -0,0 +1,130 @@
+__copyright__ = """
```

Comment:
Dumb question: should this be `pytools.graph.graphviz`?

Reply:
Dunno - the function names usually refer to dot: `show_dot`, `get_dot_code`, ...

Reply:
While `dot` is certainly the most-used of the graphviz tools, there are way more tools in the package that this could easily support.

The fact that we've made a certain mistake in the past doesn't mean that we have to keep making it. :slightly_smiling_face:

Reply:
I renamed the file to `pytools.graphviz` in [6f178ce](https://github.com/inducer/pytools/pull/151/commits/6f178cede6273cd8f33f7121a11a9efc1a15e8a3). Since `show_dot` etc. aren't really specific to `graph`, I think putting them under `pytools.graph` may not be the best fit.